src/patient-lab-details/patient-lab-details.test.tsx

```diff
@@ -334,6 +334,111 @@ describe('Patient lab details', () => {
       ).toBeInTheDocument(),
     )
   })
+
+  it('should populate based on property if a pending order is selected', async () => {
+    Object.defineProperty(window, 'localStorage', {value: localStorageMock})
+    when(usePatient)
+      .calledWith(mockPatientUuid)
+      .mockReturnValue({
+        patient: {id: mockPatientUuid},
+      })
+    localStorage.setItem('i18nextLng', 'en')
+
+    const file = new File(['content'], 'test.pdf', {type: 'application/pdf'})
+    const mockedOpenmrsFetch = openmrsFetch as jest.Mock
+    mockedOpenmrsFetch
+      .mockReturnValueOnce(mockPendingLabOrdersResponse)
+      .mockReturnValueOnce(mockLabTestsResponse)
+      .mockReturnValueOnce(mockUploadFileResponse)
+      .mockReturnValue(mockDiagnosticReportResponse)
+
+    const mockedLayout = useLayoutType as jest.Mock
```

Comment:
use `@openmrs/esm-framework/mock` so you dont need to mock most of the esm-framework functions

Reply:
updated the test file to use `@openmrs/esm-framework/mock`